lib/elixir/pages/Typespecs.md

```diff
@@ -49,13 +49,15 @@ The syntax Elixir provides for type specifications is similar to [the one in Erl
           | Literals                # Described in section "Literals"
           | Builtin                 # Described in section "Built-in types"
           | Remotes                 # Described in section "Remote types"
+          | UserDefined             # Described in section "User-defined types"
 
 ### Literals
 
 The following literals are also supported in typespecs:
 
-    type :: :atom                         ## Atoms
-          | true | false | nil            # Special atom literals
+    type ::                               ## Atoms
+            :atom                         # atoms: :foo, :bar, ...
```

Comment:
it is spaced intentionally to be aligned with the contents bellow, and not with the vertical bar.
But it's a matter of style. what ever you want.